SwiftI18n/Classes/Main/I18nManager.swift

```diff
@@ -17,10 +17,29 @@ public class I18nManager {
     public var defaultLanguage: String?
     public var availableLanguages: [String]?
     
+    /// The language code to use as a fallback if a translation is not available in the preferred language.
+    ///
+    /// **Example**:
+    /// ```swift
+    /// localizationManager.fallbackLanguage = "en_gb"
+    /// ```
+    ///
+    /// **Important Note**: When `fallbackLanguage` is set, the system will attempt to fetch translations
+    /// in this fallback language if the key matches translation in preferred language. This behavior ensures that
+    /// missing localizations are substituted with a consistent backup language.
+    ///
+    /// **Risks**:
+    /// - If a localization key is the same as the translation (e.g., common terms like "ok," "done," or "cancel"),
+    ///   and the `fallbackLanguage` is not set to English, the system will return a translation in the fallback
+    ///   language, even if an appropriate translation exists in the preferred language, this can lead to unintended substitutions
```

Comment:
```suggestion
    ///   language, even if an appropriate translation exists in the preferred language, this can lead to unintended substitutions. 
    /// To avoid having the issue in the app, use more descriptive translation keys or use case-sensitive words as a translated word (e.g., "ok" != "Ok", "done" != "Done", etc.).
```